xbmc/cores/VideoPlayer/Buffers/VideoBufferDRMPRIME.cpp

```diff
@@ -20,6 +20,18 @@ extern "C"
 namespace DRMPRIME
 {
 
+std::string GetColorimetry(const VideoPicture& picture)
+{
+  switch (picture.color_space)
+  {
+    case AVCOL_SPC_BT2020_CL:
+    case AVCOL_SPC_BT2020_NCL:
+      return "BT2020_RGB";
+  }
```

Comment:
I don't know really. Also, others  can be added in the future.

Reply:
I hardly guess that `AVCOL_SPC_BT2020_NCL`is `DRM_MODE_COLORIMETRY_BT2020_YCC` https://github.com/torvalds/linux/blob/b96fbd602d35739b5cdb49baa02048f2c41fdab1/drivers/gpu/drm/drm_connector.c#L957  and  `AVCOL_SPC_BT2020_CL` is `BT2020_CYC` https://github.com/torvalds/linux/blob/b96fbd602d35739b5cdb49baa02048f2c41fdab1/drivers/gpu/drm/drm_connector.c#L953 since https://ffmpeg.org/doxygen/4.4/pixfmt_8h.html#aff71a069509a1ad3ff54d53a1c894c85 is talking about *YUV* colorspaces